tests/components/pyload/snapshots/test_sensor.ambr

```diff
@@ -115,7 +115,7 @@
     }),
     'original_device_class': None,
     'original_icon': None,
-    'original_name': 'Downloads total',
+    'original_name': 'Finished downlods',
```

Comment:
Same here.

Reply:
thats the snapshot, it gets autogenerated

Reply:
Yeah, I noticed it kept ongoing after here.